README.md

```diff
@@ -7,6 +7,16 @@ Click this button and it will help you create a new repo, create a new Netlify p
 
 [![Deploy to Netlify Button](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/angular-quickstart)
 
+## Table of Contents:
+
+- [Setup](#setup)
+- [Deploying](#deploying)
+- [Testing](#testing)
+  - [Included Default Testing](#included-default-testing)
+  - [Removing Renovate](#removing-renovate)
+  - [Removing Cypress](#removing-cypress)
+- [Angular 💙 Netlify Resources](#angular--netlify-resources)
```

Comment:
If we want we could keep emojis in the display text of the table of contents but leave the headings the way they are now to make things map correctly.

Right now everything except this line `Angular 💙 Netlify Resources` works both on VSCode and GitHub because VSCode creates a heading link as `#angular-💙-netlify-resources` vs. GitHub makes `#angular--netlify-resources`.

Reply:
i didn't particularly like how it looked in the TOC but i'm open to whatever you and @taty2010 think. i can also take the 💙 out to make sure we're consistent with the naming links.

Reply:
I am down with whatever works best, I personally felt that the emojis in each header made it easier to separate out each section when scrolling through the README but now that we are adding the TOC I think we can live without them.

Reply:
i totally agree. i did like the visual guide to the docs 😕